scootapi/setup/sched.go

```diff
@@ -0,0 +1,62 @@
+package setup
+
+import (
+	"log"
+)
+
+// SchedulerStrategy will startup a Scheduler (or setup a connection to one)
+type SchedulerStrategy interface {
+
+	// Startup starts up a Scheduler, returing the address of the server or an error
+	Startup() (string, error)
```

Comment:
This should be StartupScheduler() for consistency. I was also thinking about using a single StartupStrategy interface and commenting that the return value is config used by upstream components, sched returns host:port and workers returns schedConfig. WDYT?